### PR TITLE
[android][file-system] fix total disk space return value

### LINKED ISSUE #6945: getFreeDiskStorageAsync & getTotalDiskCapacityAsync return the MAX_SAFE_INTEGER
### Environment

Expo CLI 3.11.7 environment info:
    System:
      OS: Windows 10
    Binaries:
      Yarn: 1.19.0 - C:\Users\fordf\AppData\Roaming\npm\yarn.CMD
      npm: 6.13.6 - C:\Program Files\nodejs\npm.CMD

Works: iOS
Broken: Android

### Steps to Reproduce

Import `expo-file-system`
Call `getFreeDiskStorageAsync()` or `getTotalDiskCapacityAsync()`
View return value of 9007199254740991

### Expected Behavior

Return either the Free Disk Storage or the Total Disk Capacity in bytes

### Actual Behavior

Returns `9007199254740991`, the MAX_SAFE_INTEGER in JavaScript

### Reproducible Demo

https://snack.expo.io/@aleef02/d16339

### PR DESCRIPTION
**Pull request by cruzach**
# Why

closes https://github.com/expo/expo/issues/6945
`getFreeDiskStorageAsync` was taken care of in https://github.com/expo/expo/pull/6465
'

### CODE REVIEW

File: packages/expo-file-system/android/src/main/java/expo/modules/filesystem/FileSystemModule.java [hunk 1]
```diff
@@ -539,7 +539,7 @@ public void getTotalDiskCapacityAsync(Promise promise) {
       long blockSize = root.getBlockSizeLong();
       BigInteger capacity = BigInteger.valueOf(blockCount).multiply(BigInteger.valueOf(blockSize));
       //cast down to avoid overflow
-      Double capacityDouble = Math.max(capacity.doubleValue(), Math.pow(2, 53) - 1);
+      Double capacityDouble = Math.min(capacity.doubleValue(), Math.pow(2, 53) - 1);
```
Comment (ALEEF02) [inline packages/expo-file-system/android/src/main/java/expo/modules/filesystem/FileSystemModule.java]: Looks good! Thanks for fixing this 👍

**Review by ALEEF02 — APPROVED** [review]
Looks good! Thanks for fixing this 👍. This should close #6945

**Review by ALEEF02 — APPROVED** [review]

File: docs/pages/versions/unversioned/sdk/filesystem.md [hunk 2]
```diff
@@ -434,6 +434,6 @@ FileSystem.getTotalDiskCapacityAsync().then(totalDiskCapacity => {
 
 #### Returns
 
-Returns a Promise that resolves to a number that specifies the total internal disk storage capacity in bytes.
+Returns a Promise that resolves to a number that specifies the total internal disk storage capacity in bytes, or Javascript's [`MAX_SAFE_INTEGER`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER) if the capacity is greater than 2<sup>53</sup> - 1 bytes.
```
Comment (sjchmiela) [inline docs/pages/versions/unversioned/sdk/filesystem.md]:
```suggestion
Returns a Promise that resolves to a number that specifies the total internal disk storage capacity in bytes, or JavaScript's [`MAX_SAFE_INTEGER`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER) if the capacity is greater than 2<sup>53</sup> - 1 bytes.
```
and in other places, nitpick

**Review by sjchmiela — APPROVED** [review]
LGTM! 🎉 Just one nitpicking comment.

**Pull request merged**